Amp.js

```diff
@@ -1,69 +1,100 @@
-'use strict';
-
+const axios = require('axios');
 const Session = require('./Session');
-const Version = require('./Version');
+const Utils = require('./Utils');
 
+const utils = new Utils();
 /**
+ *
  * Amp
  * Constructs Amp instance with passed configuration.
- *
  * @constructor
- * @param {Object} options
- *
- * Options:
- *  key - project key
- *  domain - domain of Amp server
- *  apiPath - path of API
- *  sessionTTL - TTL for session
- *  timeout - TTL for requests
+ * @param  {string} key - project key
```

Comment:
Why are you changing the interface?  The interface needs to adhered to the common spec.